.github/release.yml

```diff
@@ -14,6 +14,12 @@ changelog:
           - technical changes only
           - user facing changes
 
+    # These will require a minor or major version increment
+    - title: "Breaking changes"
+      labels:
+        - breaking-change
+        - major-breaking-change
```

Comment:
Other labels don't have dashes, just spaces. Shall we keep that consistent?

Reply:
We are not excluding these labels from other categories. So breaking changes will appear twice in the release notes. But that's okay, I think. It helps knowing which other impact like API changes the breaking change has.

Reply:
Good point. I wasn't sure which we preferred. From a developer perspective I went with dashes :)
From a ui perspective, spaces make sense.

I'm also a bit worried about the language "breaking". It sounds like we're introducing a bug. It's probably ok for the label names which are only seen by delievery team, but for release notes it would be good to use something that sounds better..

Reply:
Hm, could it also be read like _breaking_ news? "Notable changes" is probably not strong enough. Looking at our description:

  - Major: if server changes are required (eg. provision with ofn-install)
  - Minor: larger change that is irreversible (eg. migration deleting data)

We could actually introduce two new headlines for server changes and irreversible migrations but that may be too specific. Maybe we just call it *Important changes*?

Reply:
_Important changes_ would work, or we could try _Significant changes_ ?

Reply:
Yes, I like *Significant changes*.